app/post.go

```diff
@@ -189,6 +189,22 @@ func (a *App) CreatePost(post *model.Post, channel *model.Channel, triggerWebhoo
 		return nil, model.NewAppError("createPost", "api.post.create_post.town_square_read_only", nil, "", http.StatusForbidden)
 	}
 
+	var ephemeralResponse *model.Post
```

Comment:
why call it ephemeralResponse if it is a post?

Reply:
Not sure... changed it to `ephemeralPost` 😅